docs/guides/frontent-style.mdx

```diff
@@ -0,0 +1,173 @@
+## @@ -0,0 +1,176 @@
```

Comment:
This diff appears in the page: 
![image](https://user-images.githubusercontent.com/39704901/124658891-9224a080-de61-11eb-8a6b-1901f23aafb9.png)

Reply:
(as well as the frontmatter below on L3 being a bit out of order)